lib/readline.js

```diff
@@ -1326,7 +1290,7 @@ exports.isFullWidthCodePoint = isFullWidthCodePoint;
 function codePointAt(str, index) {
   var code = str.charCodeAt(index);
   var low;
-  if (0xd800 <= code && code <= 0xdbff) { // High surrogate
+  if (code >= 0xd800 && code <= 0xdbff) { // High surrogate
```

Comment:
Instead can't we just do `'no-cond-assign': 'always'` that ensures we don't do anything stupid?